src/main/java/goodgenerator/blocks/tileEntity/ExtremeHeatExchanger.java

```diff
@@ -206,9 +206,9 @@ protected GT_Multiblock_Tooltip_Builder createTooltip() {
             .addInfo(BLUE_PRINT_INFO)
             .addSeparator()
             .addController("Front bottom")
-            .addOtherStructurePart("Input Hatch", "distilled water", 1)
-            .addOtherStructurePart("Output Hatch", "SC Steam/SH Steam/Steam", 2)
-            .addOtherStructurePart("Input Hatch", "Hot fluid or plasma", 3)
+            .addOtherStructurePart("Input Hatch", "Distilled water. Any bottom left/right side casing", 1)
+            .addOtherStructurePart("Output Hatch", "SC Steam/SH Steam/Steam. Any top layer casing", 2)
+            .addOtherStructurePart("Input Hatch", "Hot fluid or plasma. Front middle on 4th layer", 3)
             .addOtherStructurePart("Output Hatch", "Cold fluid", 4)
```

Comment:
```suggestion
            .addOtherStructurePart("Output Hatch", "Cold fluid. Back middle on 4th layer", 4)
```
Just for consistency